settings.gradle

```diff
@@ -1 +1 @@
-include ':app', 'no-player'
+include ':app', ':player', 'no-player'
```

Comment:
How come copying `no-player` instead of using the lib?

Reply:
we need to patch it to allow audio playback, and by patch I mean comment out a bunch of checks and pass nulls

Reply:
which bit was the audio patch?

Reply:
all the changes to `no-player`, mostly around ignoring the if the `playerView` has been attached